### ISSUE TITLE
/build/services" does not exist in the hypothetical file system!

### ISSUE THREAD

**Issue by AhsanAyaz** [jvandemo/generator-angular2-library#105]
While creating the build, it is throwing the error mentioned doing `npm run build`.

I have this hierarchy of project: 
```
--src
----index.ts
----services
------sample-service1.service.ts
------sample-service2.service.ts
------index.ts
```
it is throwing the error then : 
```
<project>/build/services" does not exist in the hypothetical file system!
    at dneError (<project>/node_modules/rollup-plugin-hypothetical/index.js
:112:10)
    at resolveId (<project>/node_modules/rollup-plugin-hypothetical/index.j
s:90:13)
    at <project>/node_modules/rollup/dist/rollup.js:1610:32
    at process._tickCallback (internal/process/next_tick.js:109:7)
```

Any idea why this might be happening? 
I'm gonna try moving the files outside the `services` folder for now

**Comment by AhsanAyaz** [comment]
^ update. the build is throwing no errors if I keep all my files in `src` folder

**Comment by jvandemo** [comment]
Thank you for reporting. I just released v10.2.1 to fix this issue.

Feel free to re-open if you still experience problems. Thanks!
